horovod/spark/common/store.py

```diff
@@ -142,6 +144,8 @@ def _remote_attrs(self, run_id, dataset_idx):
     def create(prefix_path, *args, **kwargs):
         if HDFSStore.matches(prefix_path):
             return HDFSStore(prefix_path, *args, **kwargs)
+        elif is_databricks() and (prefix_path.startswith("dbfs:/") or prefix_path.startswith("/dbfs/")):
```

Comment:
Does Store.create accept `file:///...` URIs? If so, we should capture that as well.